Run.scala

```diff
@@ -1,7 +1,9 @@
+import scala.annotation.tailrec
+
 object Run {
    implicit class IntTimes(x: Int) {
       def times [A](f: =>A): Unit = {
```

Comment:
the scaliform should add a space before the A

Reply:
scalariform doesn't seem to do that. weird